docs/HowTo/Monitor/Elastic-Stack.md

```diff
@@ -0,0 +1,30 @@
+description: Using Elastick Stack (ELK) with Hyperledger Besu
+<!--- END of page meta data -->
+
+# Elastic Stack 
+
+Elastic Stack (also known as ELK) is a common open-source log management platform. 
+
+The [Besu Quickstart](https://github.com/PegaSysEng/besu-quickstart) provides an example implementation 
+using Elastic Stack for log management. 
+
+The [Filebeat configuration](https://github.com/PegaSysEng/besu-quickstart/blob/master/filebeat/filebeat.yml)
+ingests the logs and outputs them to Redis for storage. Using Redis enables logs to be stored and pulled 
+by Logstash as needed. Using Redis provides a highly available log management configuration. 
```

Comment:
Suggest the last 2 lines read something like this `Redis provides a highly available mechanism which enables logs to be stored by any of the Elastic Beats and pulled by Logstash as required.`

Reply:
Updated